tests/stub/session_run_parameters/test_session_run_parameters.py

```diff
@@ -116,7 +116,9 @@ def test_tx_meta(self):
             with self.subTest(routing=routing):
                 self._run("tx_meta", routing,
                           session_args=("w",),
-                          run_kwargs={"tx_meta": {"akey": "aval"}})
+                          run_kwargs={
+                              "tx_meta": {"akey": types.CypherString("aval")}
```

Comment:
If we are changing to supplying values as explicit Cypher values, we might want to update it in other places as well. For instance, Java backend does not expect this.

An example of other place: https://github.com/neo4j-drivers/testkit/blob/5.0/tests/stub/driver_parameters/test_bookmark_manager.py#L120